client/src/App.js

```diff
@@ -254,7 +254,7 @@ class App extends Component {
   };
 
   getAllUsers = () => {
-    Axios.get(`/api/users/${this.state.userObj.homeId}?isActive`).then((allUsers) => {
+    Axios.get(`/api/users/${this.state.userObj.homeId}`).then((allUsers) => {
```

Comment:
this was the issue. not sure why it went in a loop but this query was failing and kept retrying. it either takes param a `isActive=true` `isActive=false` or no `isActive` query param